R/utils.R

```diff
@@ -9,6 +9,15 @@ check_no_value_is_missing <- function(data, column) {
   invisible(data)
 }
 
+check_unique_id <- function(data, column){
+  if (sum(duplicated(data[[column]]))) {
+    abort(
+      class = "unique_ids",
+      sprintf("Column `%s` must not contain any duplicates.", column)
+    )
+  }
+}
```

Comment:
This function is called for its side effect; when the call to `if()` is FALSE we need to return `invisible(data)` (https://principles.tidyverse.org/out-invisible.html).

Reply:
Done it.